package.json

```diff
@@ -32,6 +32,7 @@
     "config": "^3.3.6",
     "env-cmd": "^10.1.0",
     "file-loader": "^6.2.0",
+    "path": "^0.12.7",
```

Comment:
Do you also need to `yarn add buffer` for the module-tutorial-client package to work?